Makefile

```diff
@@ -87,7 +87,7 @@ build:
 	make stop
 
 clean: deps
-	rm -rf $(GOPATH)/pkg/*
```

Comment:
We could just wildcard it, but I thought netplugin only compiled and ran on Linux because of the `x/sys/unix` library?